rmgpy/reactionTest.py

```diff
@@ -689,7 +691,6 @@ def testGenerateReverseRateCoefficientPDepArrhenius(self):
         """
         Test the Reaction.generateReverseRateCoefficient() method works for the PDepArrhenius format.
         """
-        from rmgpy.kinetics import PDepArrhenius
```

Comment:
There are some similar imports on line 1241, can some of these also be removed?

Reply:
Done